### PR TITLE
HTTPCORE-642: Cleaner ConnectionFactory builder ContentLengthStrategy names

### PR DESCRIPTION
**Pull request by carterkozak**

### CODE REVIEW

File: httpcore5/src/main/java/org/apache/hc/core5/http/impl/io/DefaultBHttpClientConnectionFactory.java [hunk 1]
```diff
@@ -140,12 +140,12 @@ public Builder charCodingConfig(final CharCodingConfig charCodingConfig) {
             return this;
         }
 
-        public Builder incomingContentStrategy(final ContentLengthStrategy incomingContentStrategy) {
+        public Builder incomingContentLengthStrategy(final ContentLengthStrategy incomingContentStrategy) {
```
Comment (michael-o) [inline httpcore5/src/main/java/org/apache/hc/core5/http/impl/io/DefaultBHttpClientConnectionFactory.java:143]: You forgot to rename the instance field names and argument name. They are still misleading.

Comment (carterkozak) [reply httpcore5/src/main/java/org/apache/hc/core5/http/impl/io/DefaultBHttpClientConnectionFactory.java]: Updated and resolved conflicts with the new responseOutOfOrderStrategy field.

**Comment by ok2c** [comment]
@carterkozak @michael-o I will fold this change-set into 503970518

**Comment by michael-o** [comment]
@ok2c Agreed.

**Comment by ok2c** [comment]
@carterkozak @michael-o Folded into 68812751e438dc7a52e64cf5d5078cd23f3421d7

**Comment by carterkozak** [comment]
Thanks!

**Comment by michael-o** [comment]
Very nice, thank you!

**Pull request closed without merging**